src/main/scala/scala/tools/refactoring/implementations/ClassDefOrganizeImports.scala

```diff
@@ -0,0 +1,8 @@
+package scala.tools.refactoring
+package implementations
+
+trait ClassDefOrganizeImports { self: OrganizeImports =>
```

Comment:
If implementations are split into multiple traits, they should be moved in a subpackage otherwise it is not clear what their purpose is.